explainer.md

```diff
@@ -6,58 +6,191 @@
+
+An explainer for the model-loader API can be found [here](https://github.com/webmachinelearning/model-loader/blob/master/explainer.md).
+
+### What is the right level of abstraction for the neural network operations?
+
+Neural network operations are mathematical functions. There are about a hundred standard functions universally supported in popular frameworks today e.g. convolution, matrix multiplication, various reductions, and normalizations. Additionally, some frameworks provide an even more extensive set of variants of these functions for ease of use. 
 
-### Tricky design choice N
+In designing the WebNN operations, a proposal to decompose high-level functions to the more rudimentary mathematical operations was considered, with the key benefit of having a reduced number of operations defined. However, such an approach would make the networks more verbose and harder to construct. It'll also risk losing the opportunity to leverage known optimizations for highly reusable functions in the operating system. For instance, most operating systems and modern hardware today support popular recurrent networks out of the box. By decomposing well-known functions into networks of smaller operations, their identities may be lost in the process.
 
-…
+To balance the needs of providing for future extensibility while ensuring maximum reuse and performance optimization opportunity, we chose to include both the standard functions and all the smaller operations making up the functions in the spec. For each high-level function defined, we make sure that all of its decomposed operations are also defined. This way, a newly-conceived function may be represented as a network of our decomposed operations, while a standard function can also be fully supported by the underlying platforms.
 
 ## Considered alternatives
 
 >One of the most important things you can do in your design process is to catalog the set of roads not taken. As you iterate on your design, you may find that major choices in your approach or API style will be revisited and enumerating the full space of alternatives can help you apply one (or more) of them later, may serve as a “graveyard” for u-turns in your design, and can give reviewers and potential users confidence that you’ve got your ducks in a row.
 
 ## References & acknowledgements
 
->Your design will change and be informed by many people; acknowledge them in an ongoing way! It helps build community and, as we only get by through the contributions of many, is only fair.
+>Your design will change and be informed by many people; acknowledge them in an ongoing way! It helps builder community and, as we only get by through the contributions of many, is only fair.
```

Comment:
typo: builder?

Reply:
That is the provided guideline text. I did not change it at all. 'It helps build community' doesn't sound too weird to me though.

Reply:
I'm fine with 'It helps build community'. The diff shows you changed 'build' to 'builder'. I guess it might be a typo.

Reply:
This typo is inherited from the explainer template, since fixed in there.

Also TAG in composed of humans who make typos at times 😀

Reply:
Fixed.